### PR TITLE
Add AdmissionReview models.

### PR DESCRIPTION
**Pull request by brendandburns**
Ref #1772 

cc @yue9944882

### CODE REVIEW

File: client-java-contrib/admissionreview/pom.xml [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+    <developers>
+        <developer>
+            <name>OpenAPI-Generator Contributors</name>
+            <email>team@openapitools.org</email>
+            <organization>OpenAPITools.org</organization>
+            <organizationUrl>http://openapitools.org</organizationUrl>
+        </developer>
+    </developers>
+
+    <build>
```
Comment (yue9944882) [inline client-java-contrib/admissionreview/pom.xml:34]: the whole `<build>..</build>` section can be removed, is it?

File: client-java-contrib/admissionreview/pom.xml [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+
+    <licenses>
+        <license>
+            <name>Unlicense</name>
+            <url>http://unlicense.org</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
```
Comment (yue9944882) [inline client-java-contrib/admissionreview/pom.xml:25]: can we remove the `<developer>` section? and the `<licenses>` section above as well?

File: client-java-contrib/admissionreview/pom.xml [hunk 3]
```diff
@@ -0,0 +1,282 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.kubernetes</groupId>
+    <artifactId>client-java</artifactId>
+    <packaging>jar</packaging>
+    <name>client-java</name>
```
Comment (yue9944882) [inline client-java-contrib/admissionreview/pom.xml:7]: `<name>client-java-admission-review</name>`

File: client-java-contrib/admissionreview/pom.xml [hunk 4]
```diff
@@ -0,0 +1,282 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.kubernetes</groupId>
+    <artifactId>client-java</artifactId>
```
Comment (yue9944882) [inline client-java-contrib/admissionreview/pom.xml:5]: `<artifactId>client-java-admission-review</artifactId>`

File: client-java-contrib/admissionreview/pom.xml [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+
+    <dependencies>
+        <dependency>
```
Comment (yue9944882) [inline client-java-contrib/admissionreview/pom.xml:213]: i think over half of the following dependencies can be removed by declaring the root pom as `<parent/>`, but we can sanitize this in a followup.

**Review by yue9944882 — COMMENTED** [review]
(1) can we add a `<parent/>` section in the new module and also adding it to the `<modules/>` list in the root pom on the other hand?
(2) ideally i think we should clarify the version of the local swagger spec, i.e. from which version of kube-apiserver it is downloaded? e.g. `admission-swagger-v1.20.4.json`?

File: client-java-contrib/admissionreview/pom.xml (hunk 1 above)
Comment (brendandburns) [reply client-java-contrib/admissionreview/pom.xml]: done.

File: client-java-contrib/admissionreview/pom.xml (hunk 2 above)
Comment (brendandburns) [reply client-java-contrib/admissionreview/pom.xml]: done.

File: client-java-contrib/admissionreview/pom.xml (hunk 3 above)
Comment (brendandburns) [reply client-java-contrib/admissionreview/pom.xml]: done.

File: client-java-contrib/admissionreview/pom.xml (hunk 4 above)
Comment (brendandburns) [reply client-java-contrib/admissionreview/pom.xml]: done.

File: client-java-contrib/admissionreview/pom.xml (hunk 5 above)
Comment (brendandburns) [reply client-java-contrib/admissionreview/pom.xml:17]: cleaned this up.

**Comment by brendandburns** [comment]
Apologies, I just committed the generated pom.xml without cleaning it up. Addressed all comments, please take another look.

**Review by yue9944882 — COMMENTED** [review]
/lgtm

**Pull request merged**